RadixWallet/Features/TransactionReviewFeature/TransactionReviewDapps/UnknownDappComponents+Reducer.swift

```diff
@@ -1,22 +0,0 @@
-struct UnknownDappComponents: FeatureReducer, Sendable {
```

Comment:
Reducer deleted since it is now a plain vanilla SwiftUI view, located in `InteractionReview/Destinations/UnknownDappComponents/UnknownDappComponents.swift`